source/reactnative/src/adaptive-cards.js

```diff
@@ -59,19 +61,39 @@ export default class AdaptiveCards extends React.Component {
     }
     return renderedElement;
   }
+  getAdaptiveCardConent() {
+    var adaptiveCardContent =
+      (
+        <View style={styles.container}>
+          <ScrollView>
+            {this.parsePayload()}
+          </ScrollView>
+        </View>
+      );
 
+    if (!Utils.isNullOrEmpty(this.payload.backgroundImage)) {
```

Comment:
**Nit**: try to avoid multiple returns. can assign this component to adaptiveCardContent variable and return adaptiveCardContent at end. else can be removed

Reply:
Wrapping the adaptiveCardContent with ImageBackground component based on the below condition.
```suggestion
    if (!Utils.isNullOrEmpty(this.payload.backgroundImage)) {
```